code/datums/repositories/radiation.dm

```diff
@@ -133,6 +133,7 @@ var/global/repository/radiation/radiation_repository = new()
 
 /mob/living/rad_act(var/severity)
 	if(severity)
```

Comment:
Purely cosmetic but might want to combine the two if() statements since there's nothing between them.

Reply:
Ah, alrighty.